src/analysisd/decoders/decode-xml.c

```diff
@@ -751,13 +761,14 @@ int ReadDecodeXML(const char *file)
 
         /* Validate arguments */
         if (pi->plugindecoder && (pi->regex || pi->order)) {
-            merror(DECODE_ADD, pi->name);
+            smerror(&errmsg, DECODE_ADD, pi->name);
             goto cleanup;
         }
 
         /* Add osdecoder to the list */
-        if (!OS_AddOSDecoder(pi, &os_analysisd_decoderlist_pn, &os_analysisd_decoderlist_nopn)) {
-            merror(DECODER_ERROR);
+        if (!OS_AddOSDecoder(pi, &os_analysisd_decoderlist_pn, 
```

Comment:
We can use lines of code between 80 and 120 characters long.
If you agree you can do the following change:
```c
/* Add osdecoder to the list */
        if  (!OS_AddOSDecoder(pi, &os_analysisd_decoderlist_pn, &os_analysisd_decoderlist_nopn, &errmsg))  {
            smerror(&errmsg, DECODER_ERROR);
            goto cleanup;
        }
```